tool/lib/commands/pub_get.dart

```diff
@@ -37,14 +47,21 @@ class PackagesGetCommand extends Command {
     final repo = DevToolsRepo.getInstance()!;
     final packages = repo.getPackages();
 
-    final upgrade = argResults!['upgrade'];
+    final upgrade = argResults![_upgradeFlag];
+    final onlyMainPackages = argResults![_onlyMainFlag];
     final command = upgrade ? 'upgrade' : 'get';
 
     log.stdout('Running flutter pub $command...');
 
     int failureCount = 0;
 
     for (Package p in packages) {
+      final packagePathParts = p.relativePath.split('/');
```

Comment:
This may fail on different OS'. Perhaps it might be better to use the [path package's split function](https://api.flutter.dev/flutter/package-path_path/split.html). It uses the operating system's path separator to do the split